src/main/java/github/pitbox46/epicfightnbt/Config.java

```diff
@@ -35,88 +36,65 @@
-    public static final Map<String, Map<String, String>> JSON_MAP = new HashMap<>();
+    public static NBTCategorySchema JSON_MAP;
 
     public static void init(Path folder) {
         File file = new File(FileUtils.getOrCreateDirectory(folder, "serverconfig").toFile(), "epicfightnbt.json");
         try {
             if (file.createNewFile()) {
-                Path defaultConfigPath = FMLPaths.GAMEDIR.get().resolve(FMLConfig.defaultConfigPath()).resolve("epicfightnbt.json");
-                if (Files.exists(defaultConfigPath)) {
-                    Files.copy(defaultConfigPath, file.toPath(), StandardCopyOption.REPLACE_EXISTING);
-                } else {
-                    FileWriter writer = new FileWriter(file);
-
-                    StringBuilder values = new StringBuilder();
-                    for (String value : DICTIONARY.keySet()) {
-                        values.append(value).append(", ");
-                    }
-                    String json = "{\n" +
-                            "  \"__comment\": \"Values: " + values + "\",\n" +
-                            "  \"double/head_left\": {\n" +
-                            "    \"double/basic_hammer_left\": \"axe\",\n" +
-                            "    \"double/basic_axe_left\": \"axe\",\n" +
-                            "    \"double/basic_pickaxe_left\": \"pickaxe\",\n" +
-                            "    \"double/hoe_left\": \"hoe\",\n" +
-                            "    \"double/claw_left\": \"axe\",\n" +
-                            "    \"double/adze_left\": \"axe\",\n" +
-                            "    \"double/sickle_left\": \"axe\"\n" +
-                            "  },\n" +
-                            "  \"sword/blade\": {\n" +
-                            "    \"sword/basic_blade\": \"sword\",\n" +
-                            "    \"sword/heavy_blade\": \"greatsword\",\n" +
-                            "    \"sword/machete\": \"tachi\",\n" +
-                            "    \"sword/short_blade\": \"dagger\"\n" +
-                            "  },\n" +
-                            "  \"single/head\": {\n" +
-                            "    \"single/spearhead\": \"spear\",\n" +
-                            "    \"single/basic_shovel\": \"shovel\"\n" +
-                            "  },\n" +
-                            "  \"crossbow/stock\": {\n" +
-                            "    \"crossbow/basic_stock\": \"crossbow\"\n" +
-                            "  },\n" +
-                            "  \"bow/string\": {\n" +
-                            "    \"bow/basic_string\": \"bow\"\n" +
-                            "  }\n" +
-                            "}";
-                    writer.write(json);
-                    writer.close();
+                InputStreamReader defaults = new InputStreamReader(Objects.requireNonNull(Thread.currentThread().getContextClassLoader().getResourceAsStream("assets/epicfightnbt/epicfightnbt.json")));
```

Comment:
You got rid of the default config. The default config works like a regular Forge default server config where the config is applied to any new world that is created. Without this, players would have to copy paste their customized config for every new world. 

(I have not tested this code)
```suggestion
                InputStreamReader defaults = null;
                Path defaultConfigPath = FMLPaths.GAMEDIR.get().resolve(FMLConfig.defaultConfigPath()).resolve("epicfightnbt.json");
                if (!Files.exists(defaultConfigPath)) {
                    Files.copy(Objects.requireNonNull(Thread.currentThread().getContextClassLoader().getResourceAsStream("assets/epicfightnbt/epicfightnbt.json")), defaultConfigPath);
                }
                defaults = new InputStreamReader(Files.newInputStream(defaultConfigPath));
```

Reply:
Oh, sorry, I didn't mean to, I'm still stuck to 1.12 and don't know all the new cool features of later versions. Re-implementing that ASAP.